README.md

```diff
@@ -3126,15 +3126,31 @@ no parameters.
 ## Exceptions
 
 * <a name="prefer-raise-over-fail"></a>
-  Prefer `raise` over `fail` for exceptions.
+  Use `raise` over `fail` to re-throw exceptions.
   <sup>[[link](#prefer-raise-over-fail)]</sup>
 
   ```ruby
   # bad
-  fail SomeException, 'message'
+  rescue => exception
+    fail exception
+  end
 
   # good
+  rescue => exception
+    raise exception
```

Comment:
We should never rethrow exceptions by using `raise exception`. Rethrows should always be simply `raise`.

The reason why is that `raise exception` will destroy the backtrace of the original exception, thereby shadowing the actual origin of the problem.

Reply:
Oh right good point.